docs/use_case/post_demo_app_intro.md

```diff
@@ -0,0 +1,53 @@
+In this tutorial we are going to present 3 versions of ONEm's #post application. The apps are written using the Django framework ONEm's Python SDK.
+
+- **postv1** - a stripped down version of the service; we will be able to create, store and view posts
+- **postv2** - on top of v1, we will now add a Search wizard so we can search by keyword/code in all saved posts (ours or others)
+- **postv3** - on top of v2, we will enable interactions with ONEm's restdAPI
+
+For prerequisites and running any of the 3 apps locally please refer to: [prerequisites](/getting_started/python_prereq) and [run locally](/getting_started/python_run_local/) tutorials.
+
+
+Notes: 
+
+1) The project containing these 3 apps can be cloned from here: [**post-demo-app**](https://github.com/ONEmCommunications/post-demo-app)
+or
+```bash
+onem@local:~$ git clone https://github.com/ONEmCommunications/post-demo-app
+```
+
+2) We will only need to create one virtualenv, containing the packages listed in the requirements.txt file in the project's root folder and this virtual env will be used by all 3 versions of the app.
+
+All 3 versions of the #post application contained in this repo(**postv1**, **v2** & **v3**) are Django apps. Below you can find a few details on the structure of a Django project, its key files and their purpose.
```

Comment:
Why don't we keep shorter lines of, let's say, max 80 chars? It's more difficult to read and edit very long lines like this.

Reply:
I agree we should make it neat and easy to read, but this does not mean a line should have 80 chars max. We are not writing code here, though PEP8 recommends a 79 char limit :smile: 

I'd suggest we keep paragraphs as a one liner. Then split them up by two new lines ..